src/css/Style.css

```diff
@@ -1,10 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
 body {
-  min-height: 100vh;
-  font-size: 1rem;
+  background-color: #f5f6fa;
 }
```

Comment:
### Your Design on 80% screen size
![design-on-80%](https://user-images.githubusercontent.com/93039359/187503342-50106716-49f1-45f9-921d-65f29d5cb08d.png)

### Your Design on 100% screen size
![student-design (copy)](https://user-images.githubusercontent.com/93039359/187503356-1169fba7-9c84-4835-b3df-edc33def0fd4.png)

- [ ] You can see in both images that on 80% screen size your design looks perfect :+1: but on 100% screen size it messes up. I know that this project is not a responsive project but please add enough responsiveness in your design to fit on all the computer screens :thinking:

**Hint: For this kind of responsiveness you just need to make widths in percentage(%) instead of pixels(px) :blush:**